tests/testthat/navigate-test.R

```diff
@@ -0,0 +1,9 @@
+path <- line_profile(eval(parse(text = "
```

Comment:
Would you mind adding a note about the use of parse? (i.e. needed to get line numbers)